options/locale/locale_en-US.ini

```diff
@@ -1974,6 +1974,12 @@ settings.lfs_pointers.inRepo=In Repo
 settings.lfs_pointers.exists=Exists in store
 settings.lfs_pointers.accessible=Accessible to User
 settings.lfs_pointers.associateAccessible=Associate accessible %d OIDs
+settings.rename_branch_failed_exist=can not rename branch because target branch %s is exist
+settings.rename_branch_failed_not_exist= can not rename branch because target branch %s is not exist
+settings.rename_branch_success = branch %s has been succed renamed to %s
+settings.rename_branch_from=old branch name
+settings.rename_branch_to=new branch name
+settings.rename_branch = Rename branch
```

Comment:
```suggestion
settings.rename_branch_failed_exist=Cannot rename branch because target branch %s exists.
settings.rename_branch_failed_not_exist=Cannot rename branch %s because it does not exist.
settings.rename_branch_success =Branch %s was successfully renamed to %s.
settings.rename_branch_from=old branch name
settings.rename_branch_to=new branch name
settings.rename_branch=Rename branch
```